styleguide.config.js

```diff
@@ -2,9 +2,17 @@
 const glob = require('glob');
 const path = require('path');
 
+const firstLetterIsUppercase = (string) => {
+    const first = string.charAt(0);
+    return first === first.toUpperCase();
+};
+
 module.exports = { // eslint-disable-line
     components: function() {
-        const folders = glob.sync('./src/Sulu/Bundle/*/Resources/js/components/*');
+        let folders = glob.sync('./src/Sulu/Bundle/*/Resources/js/components/*');
+
+        // filter out higher order components
```

Comment:
Do we really want that? Could be useful for others to know how to use this kind of components. Actually I feel like this is even more important for these components than for others...